bevy_rapier3d/examples/ray_casting3.rs

```diff
@@ -92,13 +92,16 @@ pub fn cast_ray(
         };
 
         // Then cast the ray.
-        let hit = rapier_context.cast_ray(
-            ray.origin,
-            ray.direction.into(),
-            f32::MAX,
-            true,
-            QueryFilter::only_dynamic(),
-        );
+        let hit = rapier_context
+            .cast_ray(
+                DEFAULT_WORLD_ID,
+                ray.origin,
+                *ray.direction,
+                f32::MAX,
+                true,
+                QueryFilter::only_dynamic(),
+            )
+            .expect("Default world should exist.");
```

Comment:
This user facing API change is unfortunate, I'd like to discuss if alternatives could be achievable, more details in main PR comment feed.